src/bespokelabs/curator/request_processor/base_online_request_processor.py

```diff
@@ -22,8 +22,9 @@
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)
 
-DEFAULT_REQUESTS_PER_MINUTE = 100
-DEFAULT_TOKENS_PER_MINUTE = 100_000
+DEFAULT_MAX_REQUESTS_PER_MINUTE = 100
```

Comment:
Why is this renamed?
Does it mean that we may do less than max_reqeusts?

Reply:
Just to be clearer. Everywhere else we are calling the rate limit max_requests_per_minute.